hummingbot/market/liquid/liquid_market.pyx

```diff
@@ -0,0 +1,1346 @@
+        order = self._in_flight_orders.get(client_order_id)
+        exchange_order_id = await order.get_exchange_order_id()
+        path_url = Constants.LIST_ORDER_URI.format(exchange_order_id=exchange_order_id)
+        result = await self._api_request("get", path_url=path_url)
+        return result
+
+    async def list_orders(self) -> List[Any]:
+        """
+        Gets a list of the user's active orders via rest API
+        :returns: json response
+        """
+        path_url = Constants.LIST_ORDERS_URI
+        result = await self._api_request("get", path_url=path_url)
+        return result
+
+    async def get_transfers(self) -> Dict[str, Any]:
+        """
+        Gets a list of the user's transfers via rest API
+        :returns: json response
+        """
+        # TODO: figure out what this transfer is
+        path_url = "/transfers"
+        result = await self._api_request("get", path_url=path_url)
+        return result
+
+    async def list_liquid_accounts(self) -> Dict[str, str]:
+        """
+        Gets a list of the user's liquid accounts via rest API
+        :returns: json response
+        """
+        accounts_dict = {}
+
+        crypto_accounts_path = Constants.CRYPTO_ACCOUNTS_URI
+        crypto_accounts = await self._api_request("get", path_url=crypto_accounts_path) or []
+
+        for account in crypto_accounts:
+            accounts_dict[account['currency']] = account
+        return accounts_dict
+
+    async def get_deposit_address(self, asset: str) -> str:
+        """
+        Gets a list of the user's crypto address for a particular asset,
+        so that the bot can deposit funds into liquid
+        :returns: json response
+        """
+        liquid_accounts = await self.list_liquid_accounts()
+        account = liquid_accounts.get(asset, {})
+        return account.get('address'), account
+
+    async def get_deposit_info(self, asset: str) -> DepositInfo:
+        """
+        Calls `self.get_deposit_address` and format the response into a DepositInfo instance
+        :returns: a DepositInfo instance
+        """
+        address, extras = await self.get_deposit_address(asset)
+        return DepositInfo(address=address, extras=extras)
+
+    async def execute_withdraw(self, str tracking_id, str to_address, str currency, object amount):
+        """
+        Function that makes API request to withdraw funds
+        Request payload example:
+        {
+            "auth_code":"0000",  # Optional
+            "crypto_withdrawal":{
+                "currency":"BTC",
+                "amount":"1",
+                "address":"1BvBMSEYstWetqTFn5Au4m4GFg7xJaNVN2",
+                "payment_id": null,
+                "memo_type": null,
+                "memo_value": null
+            }
+        }
+        """
+        path_url = Constants.CRYPTO_WITHDRAWAL_URI
+
+        data = {
+            "crypto_withdrawal": {
+                "currency": currency,
+                "amount": str(amount),
```

Comment:
You might want to look into checking if the amount specified satisfies the minimum withdrawal amount on Liquid